docs/dev/i18n.rst

```diff
@@ -4,8 +4,8 @@ Internationalization support
 General guidelines
 ------------------
 
-* All config mode packages must be fully translatable, with complete English and German texts.
-* All new expert mode packages be fully translatable. English texts are required, German texts recommended.
+* All config mode packages must be fully translatable, with complete English and German texts, French texts recommended
+* All new expert mode packages be fully translatable. English texts are required, German and French texts recommended.
```

Comment:
Maybe: "Translations for other supported languages are nice-to-have, but not required. If you don't know a language well, rather leave the translation blank, so it is obvious that there is no proper translation yet."